<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QSmartTools</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>50</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>QSmartTools</string>
      </property>
      <property name="textFormat">
       <enum>Qt::RichText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <property name="indent">
       <number>3</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>679</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPointTranslate"/>
   <addaction name="actionRename"/>
   <addaction name="actionTcpTest"/>
   <addaction name="actionIphoneToiTouch"/>
   <addaction name="actionIphoneToiPad"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionPointTranslate">
   <property name="text">
    <string>pointTranslate</string>
   </property>
  </action>
  <action name="actionRename">
   <property name="text">
    <string>rename</string>
   </property>
  </action>
  <action name="actionIphoneToiTouch">
   <property name="text">
    <string>iphoneToiTouch</string>
   </property>
  </action>
  <action name="actionIphoneToiPad">
   <property name="text">
    <string>iphoneToiPad</string>
   </property>
  </action>
  <action name="actionTcpTest">
   <property name="text">
    <string>tcpTest</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
